ui/components/component-library/icon/icon.scss

```
@@ -39,4 +39,8 @@
   &--size-xl {
     --size: 32px;
   }
+
+  &--size-auto {
+    --size: 1em;
```

I know this does the same thing but I wonder if `inherit` would be more descriptive of what it is doing?

```suggestion
    --size: inherit; // Inherits parent font-size
```

---

Ahh ok I must have updated after I was looking and made a mistake my bad! Still think it's worth adding the description but yup right the first time apologies!